internal/operands/metrics/reconcile_test.go

```diff
@@ -54,14 +56,51 @@ var _ = Describe("Metrics operand", func() {
 			Logger:       log,
 			VersionCache: common.VersionCache{},
 		}
+	})
+
+	AfterEach(func() {
+		os.Unsetenv(runbookURLTemplateEnv)
+	})
 
+	It("should create metrics resources", func() {
 		_, err := operand.Reconcile(&request)
 		Expect(err).ToNot(HaveOccurred())
 		ExpectResourceExists(newPrometheusRule(namespace), request)
 		ExpectResourceExists(newServiceMonitorCR(namespace), request)
 		ExpectResourceExists(newMonitoringClusterRole(), request)
 		ExpectResourceExists(newMonitoringClusterRoleBinding(), request)
 	})
+
+	It("should use the default runbook URL template when no ENV Variable is set", func() {
```

Comment:
You might be able to deduce both tests into one `DescribeTable`. The only difference between both tests is whether the env variable was set or not.

Reply:
Done. Also added a case where an invalid value is set for the env variable.